resources/todomvc/vanilla-examples/javascript-es5/complex/scripts/build.js

```diff
@@ -0,0 +1,79 @@
+const fs = require("fs").promises;
+const { JSDOM } = require("jsdom");
+const path = require("path");
+const { getHtmlContent } = require("big-dom-generator/utils/getHtmlContent");
+
+const ROOT_DIRECTORY = "./";
+const SOURCE_DIRECTORY = "./shared/src";
+const TARGET_DIRECTORY = "./complex/dist";
+
+const COMPLEX_DOM_HTML_FILE = "index.html";
+const TODO_HTML_FILE = "index.html";
+
+const FILES_TO_MOVE = ["node_modules/todomvc-common/base.css", "node_modules/todomvc-app-css/index.css", "node_modules/big-dom-generator/dist/app.css", "node_modules/big-dom-generator/generated.css", "node_modules/big-dom-generator/dist/logo.png"];
+
+const CSS_FILES_TO_ADD_LINKS_FOR = ["app.css", "generated.css"];
+
+async function build() {
+    // remove dist directory if it exists
+    await fs.rm(path.resolve(TARGET_DIRECTORY), { recursive: true, force: true });
+
+    // re-create the directory.
+    await fs.mkdir(path.resolve(TARGET_DIRECTORY));
+
+    // copy src folder
+    await fs.cp(path.resolve(SOURCE_DIRECTORY), path.resolve(TARGET_DIRECTORY), { recursive: true }, (err) => {
+        if (err)
+            console.error(err);
+    });
+
+    // copy files to move
+    for (let i = 0; i < FILES_TO_MOVE.length; i++)
+        await fs.copyFile(path.resolve(__dirname, "../../", FILES_TO_MOVE[i]), path.join(TARGET_DIRECTORY, path.basename(FILES_TO_MOVE[i])));
+
+    // read todo.html file
+    let html = await fs.readFile(path.join(ROOT_DIRECTORY, TODO_HTML_FILE), "utf8");
+
+    // remove base paths from files to move
+    for (let i = 0; i < FILES_TO_MOVE.length; i++)
+        html = html.replace(FILES_TO_MOVE[i], path.basename(FILES_TO_MOVE[i]));
+
```

Comment:
Let me know if I don't get the whole script, but could we just copy over the dist folder from the standalone version and make the updates to the index.html file that are needed?

Reply:
Yes, that's correct. I've also extracted this script to big-dom-generator/utils/buildComplex.js so it can be shared by the architectures.